processing/src/main/java/org/apache/druid/segment/data/BlockLayoutColumnarDoublesSupplier.java

```diff
@@ -168,7 +168,7 @@ public void get(final double[] out, final int[] indexes, final int length)
 
     protected void loadBuffer(int bufferNum)
     {
-      CloseQuietly.close(holder);
+      CloseableUtils.closeAndWrapExceptions(holder);
```

Comment:
This seems fine because `ResourceHolder.close()` doesn't throw `IOException`, but this code seems confusing to me since, until looking at the code closely, I wasn't sure if it's fine to not execute the below lines when this line throws an exception. How about adding a comment here, such as "this line should never throw any checked exceptions"?

Reply:
Same comment for other places where close `ResourceHolder`s.

Reply:
I didn't realize it didn't throw `IOException`. So actually, I could just remove the CloseableUtils call completely here,  and replace it with `holder.close()`. I'll do that.

Reply:
Did that, along with the other block layout column suppliers.